src/erlcloud_ddb2.erl

```diff
@@ -1140,6 +1148,8 @@ out({error, Reason}, _, _) ->
     {error, Reason};
 out(ok, _, _) ->
     {error, unexpected_empty_response};
+out({ok, {request, _Headers, _Body}} = Request, _Undynamize, _Opts) ->
```

Comment:
It looks like the spec for `out/3` is broken by this, because this form is not documented.

I think you also need to cover this one in `ddb_return/2` type spec.

Reply:
After the rebase we should see this in the Action results 😄